modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -91,6 +91,25 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     });
   };
 
+  const toggleCaption = () => {
+    getSelectionStartCellOrCaption(editor).each((cellOrCaption) => {
+      TableLookup.table(cellOrCaption, isRoot).each((table) => {
+        const captionElementOpt = Selectors.one('caption', table);
+
+        captionElementOpt.fold(
+          () => {
+            const caption = SugarElement.fromTag('caption');
+            Insert.append(caption, SugarElement.fromText('Caption'));
+            Insert.appendAt(table, caption, 0);
```

Comment:
This is inconsistent behaviour with how the dialog works, as it does not insert the "Caption" text. It should be the same and really the dialog should be using this command.

- [ ] Verify with design if the "Caption" text should be getting inserted
- [ ] Make the table dialog use the command so all behaviour is consitent

Reply:
The word "Caption" should be inserted. Ideally that shows up when using the dialog as well. This is to make it consistent with image captions, and also to show the user where the table captions are 👀.

Reply:
It should definitively come with the "Caption"-text as there is an issue to add that...